### ISSUE TITLE
[Bug] screen readers do not indicate if an effect is disabled or enabled

### ISSUE THREAD

**Issue by marrie** [crowbartools/Firebot#2218]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

when clicking the drop down menu by effects, screen readers cannot tell when an effect is disabled or enabled. The link just says "toggle enabled."

### Expected Behavior

I want to be able to see if an effect is enabled or disabled. If I need to disable for troubleshooting purposes this makes a bit hard for me as I cannot keep in my head  what is enabled and what is disabled.

### Steps To Reproduce

1. for example, click "events
2. click the drop down menu then click edit.
3. click the dropdown menu next to any effect. Notice that the screen reader will not say or show if an effect is disabled or enabled. This is everywhere a dropdown menu for editing effects is located.  I'mnot sure what it shows visually, however if this could be conveyed to the screen reader that would be wonderful.

### Version

Firebot v5.57.0

### What operating system are you using Firebot on?

Windows 10

### Contact Details (Optional)

discord: ke7zum
